src/deprecated.jl

```diff
@@ -19,21 +19,6 @@ function exists end
 ### Changed in PR #844
 @deprecate silence_errors(f::Function) f()
 
-### Changed in PR #845
```

Comment:
I decided to delete this since it was never on  a tagged release and we just merged these changes in the last week. So no one on a tagged version would be effected by this change.